<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BalanceProofDialog</class>
 <widget class="QDialog" name="BalanceProofDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Proof of balance</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="11" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>7</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="m_balance">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Balance </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_addressLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="m_signatureEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Signature</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="m_proofHorizontalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="m_copyProofButton">
         <property name="text">
          <string>Copy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_saveProofButton">
         <property name="text">
          <string>Save to file</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_proofCloseButton">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_copyProofButton</tabstop>
  <tabstop>m_saveProofButton</tabstop>
  <tabstop>m_proofCloseButton</tabstop>
 </tabstops>
 <connections>
  <connection>
   <sender>m_proofCloseButton</sender>
   <signal>clicked()</signal>
   <receiver>BalanceProofDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>755</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_copyProofButton</sender>
   <signal>clicked()</signal>
   <receiver>BalanceProofDialog</receiver>
   <slot>copyProof()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_saveProofButton</sender>
   <signal>clicked()</signal>
   <receiver>BalanceProofDialog</receiver>
   <slot>saveProof()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>copyProof()</slot>
  <slot>saveProof()</slot>
 </slots>
</ui>
